<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scenery3dDialogForm</class>
 <widget class="QWidget" name="scenery3dDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>550</height>
   </rect>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBar" name="titleBar">
     <property name="title">
      <string>3D Scenery</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="viewContent">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="2">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QTextBrowser" name="scenery3dTextBrowser">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="html">
              <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="openExternalLinks">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="2">
            <widget class="QListWidget" name="scenery3dListWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QCheckBox" name="checkBoxDefaultScene">
                 <property name="text">
                  <string>Set as default scene</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonOpenStoredViewDialog">
                 <property name="text">
                  <string>Load/save viewpoints</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTabWidget" name="tabWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="tab_VisualOptions">
                <attribute name="title">
                 <string>Visual options</string>
                </attribute>
                <layout class="QFormLayout" name="formLayout_4">
                 <property name="fieldGrowthPolicy">
                  <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                 </property>
                 <item row="0" column="0" colspan="2">
                  <widget class="QCheckBox" name="checkBoxTorchlight">
                   <property name="text">
                    <string>Torchlight (%1)</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="labelTorchStrength">
                   <property name="text">
                    <string>Torch strength:</string>
                   </property>
                   <property name="buddy">
                    <cstring>sliderTorchStrength</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="labelTorchRange">
                   <property name="text">
                    <string>Torch range:</string>
                   </property>
                   <property name="buddy">
                    <cstring>sliderTorchRange</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QSlider" name="sliderTorchRange">
                   <property name="maximum">
                    <number>10000</number>
                   </property>
                   <property name="singleStep">
                    <number>100</number>
                   </property>
                   <property name="pageStep">
                    <number>1000</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="labelDirectionalLightPush">
                   <property name="text">
                    <string>Directional light enhancement</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QDoubleSpinBox" name="directionalLightPushSpinBox">
                   <property name="toolTip">
                    <string>Increase the power of directional light, e.g. to enhance contrast for patches of light cast through small holes.</string>
                   </property>
                   <property name="minimum">
                    <double>0.200000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>20.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.050000000000000</double>
                   </property>
                   <property name="value">
                    <double>1.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QSlider" name="sliderTorchStrength">
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="QGroupBox" name="gridCoordinatesGroupBox">
                   <property name="title">
                    <string>Show grid coordinates (%1)</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QRadioButton" name="radioButton_GCTopRight">
                      <property name="toolTip">
                       <string>Show coordinates in the top right screen corner</string>
                      </property>
                      <property name="text">
                       <string>Top Right</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="radioButton_GCBottom">
                      <property name="toolTip">
                       <string>Show coordinates centered in the bottom of the screen</string>
                      </property>
                      <property name="text">
                       <string>Bottom Center</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="QCheckBox" name="overdrawLandscapePolylineCheckBox">
                   <property name="toolTip">
                    <string>Overdraw horizon polygon defined in your landscape. This displays the inaccuracies in a very large planar model compared to Earth curvature, of course only for a single point of observation.</string>
                   </property>
                   <property name="text">
                    <string>Draw horizon polyline in foreground</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="0">
                  <widget class="QCheckBox" name="ignoreInitialViewCheckBox">
                   <property name="toolTip">
                    <string>Do no use the configured direction and field of view. This may be helpful in a planetarium environment.</string>
                   </property>
                   <property name="text">
                    <string>Ignore initial view settings</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_QualityPerformanceOptions">
                <attribute name="title">
                 <string>Quality/Performance settings</string>
                </attribute>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <layout class="QFormLayout" name="formLayout">
                   <property name="fieldGrowthPolicy">
                    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                   </property>
                   <item row="0" column="0" colspan="2">
                    <widget class="QCheckBox" name="checkBoxEnablePixelLight">
                     <property name="text">
                      <string>Per-Pixel lighting</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0" colspan="2">
                    <widget class="QCheckBox" name="checkBoxEnableShadows">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="text">
                      <string>Shadows</string>
                     </property>
                     <property name="checked">
                      <bool>false</bool>
                     </property>
                     <property name="tristate">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0" colspan="2">
                    <layout class="QFormLayout" name="formLayout_6">
                     <property name="leftMargin">
                      <number>20</number>
                     </property>
                     <item row="0" column="0" colspan="2">
                      <widget class="QCheckBox" name="checkBoxSimpleShadows">
                       <property name="text">
                        <string>Simple shadows</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item row="3" column="0" colspan="2">
                    <widget class="QCheckBox" name="checkBoxEnableBump">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="text">
                      <string>Bump mapping</string>
                     </property>
                     <property name="checked">
                      <bool>false</bool>
                     </property>
                     <property name="tristate">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item row="4" column="0">
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>Cubemap size:</string>
                     </property>
                     <property name="buddy">
                      <cstring>comboBoxCubemapSize</cstring>
                     </property>
                    </widget>
                   </item>
                   <item row="4" column="1">
                    <widget class="QComboBox" name="comboBoxCubemapSize"/>
                   </item>
                   <item row="5" column="0">
                    <widget class="QLabel" name="labelShadowmapSize">
                     <property name="text">
                      <string>Shadowmap size:</string>
                     </property>
                     <property name="buddy">
                      <cstring>comboBoxShadowmapSize</cstring>
                     </property>
                    </widget>
                   </item>
                   <item row="5" column="1">
                    <widget class="QComboBox" name="comboBoxShadowmapSize">
                     <property name="currentIndex">
                      <number>-1</number>
                     </property>
                    </widget>
                   </item>
                   <item row="6" column="0">
                    <widget class="QLabel" name="labelFilterQuality">
                     <property name="text">
                      <string>Filtering Quality:</string>
                     </property>
                     <property name="buddy">
                      <cstring>comboBoxShadowFiltering</cstring>
                     </property>
                    </widget>
                   </item>
                   <item row="6" column="1">
                    <widget class="QComboBox" name="comboBoxShadowFiltering">
                     <item>
                      <property name="text">
                       <string>Off</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Hardware</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Low</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Low + Hardware</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>High (slow)</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>High + Hardware (slow)</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="7" column="0" colspan="2">
                    <layout class="QFormLayout" name="formLayout_5">
                     <property name="leftMargin">
                      <number>20</number>
                     </property>
                     <item row="0" column="0" colspan="2">
                      <widget class="QCheckBox" name="checkBoxPCSS">
                       <property name="text">
                        <string>PCSS</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QFormLayout" name="formLayout_2">
                   <property name="fieldGrowthPolicy">
                    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>Cubemapping mode:</string>
                     </property>
                     <property name="buddy">
                      <cstring>comboBoxCubemapMode</cstring>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="comboBoxCubemapMode"/>
                   </item>
                   <item row="1" column="0" colspan="2">
                    <widget class="QCheckBox" name="checkBoxCubemapShadows">
                     <property name="text">
                      <string>Full cubemap shadows</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0" colspan="2">
                    <widget class="QCheckBox" name="checkBoxEnableLazyDrawing">
                     <property name="text">
                      <string>Lazy cubemap re-rendering</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="0" colspan="2">
                    <layout class="QFormLayout" name="formLayout_3">
                     <property name="leftMargin">
                      <number>20</number>
                     </property>
                     <item row="0" column="0">
                      <widget class="QLabel" name="labelLazyDrawingInterval">
                       <property name="text">
                        <string>Interval:</string>
                       </property>
                       <property name="buddy">
                        <cstring>spinLazyDrawingInterval</cstring>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QDoubleSpinBox" name="spinLazyDrawingInterval">
                       <property name="minimum">
                        <double>0.100000000000000</double>
                       </property>
                       <property name="maximum">
                        <double>600.000000000000000</double>
                       </property>
                       <property name="singleStep">
                        <double>0.100000000000000</double>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0" colspan="2">
                      <widget class="QCheckBox" name="checkBoxDominantFace">
                       <property name="text">
                        <string>Only update dominant face when moving</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0" colspan="2">
                      <widget class="QCheckBox" name="checkBoxSecondDominantFace">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="text">
                        <string>Also update second-most dominant face</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StelCloseButton</class>
   <extends>QPushButton</extends>
   <header>gui/StelCloseButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
